composer.json

```diff
@@ -56,15 +56,13 @@
     "config": {
         "sort-packages": true
     },
-    "extra": {
-        "branch-alias": {
```

Comment:
Why do we need to remove branch alias?

Reply:
Because https://github.com/a2lix/TranslationFormBundle/blob/master/composer.json#L62 should be the only one IMO.

Reply:
But, for example: https://github.com/symfony/symfony/blob/3.4/composer.json

Symfony adds it on all branches.

Isnt it necessary to use the 2.x-dev alias?

Reply:
I don't think so. It's true that Symfony and Api-Platform keep this alias in their composer but from what I understand, it has only usability for master branch.
And some people, remove it https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3555 in not master branch.